scripts/storage-emulator-integration/conformance/firebase.endpoints.test.ts

```diff
@@ -173,6 +173,37 @@ describe("Firebase Storage endpoint conformance tests", () => {
           })
           .expect(403);
       });
+
+      it("should return 403 when resumable upload is unauthenticated and finalize is called again", async () => {
```

Comment:
Pointed tests against prod as well and now it passes on both the emulator and prod.